site/app/models/Config.php

```diff
@@ -167,6 +168,8 @@ class Config extends AbstractModel {
     protected $forum_enabled;
     /** @property @var bool */
     protected $regrade_enabled;
+    /** @property @var bool */
+    protected $manual_grading_enabled;
```

Comment:
Why is this a course-wide setting?

Reply:
When the manual_grading_enabled option is unselected, the verify button is hidden (so no more verification). I vaguely remember Barb mentioning turning on and off verifying: I'll run it by her when she gets back. Also I renamed "manual grading enabled" to "verify enabled"